### PR TITLE
Fixes #1236 Create tests for organization_feed.dart

### LINKED ISSUE #1236: Views: Create tests for organization_feed.dart
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/views/after_auth_screens/feed/organization_feed.dart`

- When complete, all methods, classes and/or functions in the file will need to be tested. 
- These tests must be placed in a single file with the name `test/views/after_auth_screens/feed/organization_feed_test.dart`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa/tree/develop/lib/views/after_auth_screens/feed/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)
- No functionality must be removed to achieve this goal

### PR DESCRIPTION
**Pull request by Ayush0Chaudhary**
**What kind of change does this PR introduce?**
[Fixes](https://github.com/PalisadoesFoundation/talawa/commit/52a881a4f64531171c982247e525f7ece9e2c91d) https://github.com/PalisadoesFoundation/talawa/issues/1236 [Create tests for organization_feed.dart](https://github.com/PalisadoesFoundation/talawa/commit/52a881a4f64531171c982247e525f7ece9e2c91d)

**Issue Number:**

Fixes #1236

**Did you add tests for your changes?**

**Summary**
[Fixes](https://github.com/PalisadoesFoundation/talawa/commit/52a881a4f64531171c982247e525f7ece9e2c91d) https://github.com/PalisadoesFoundation/talawa/issues/1236 [Create tests for organization_feed.dart](https://github.com/PalisadoesFoundation/talawa/commit/52a881a4f64531171c982247e525f7ece9e2c91d)

**Does this PR introduce a breaking change?**
NO

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
YES

### CODE REVIEW

**Comment by Ayush0Chaudhary** [comment]
I achieved **86.5%** coverage

Screenshot from 2023-02-09 17-44-20

Screen Shot for uncovered part
Screenshot from 2023-02-09 17-46-17

@palisadoes @noman2002 @TheHazeEffect

**Comment by literalEval** [comment]
@Ayush0Chaudhary do you need help in getting to 100% ?

**Comment by palisadoes** [comment]
1. We are going to have delay any potential merge until this PR is complete https://github.com/PalisadoesFoundation/talawa/pull/1474
1. It may cause minor merge conflicts, so be prepared

**Review by noman2002 — CHANGES_REQUESTED** [review]
Please try to get it to 100%

**Comment by palisadoes** [comment]
Please fix the linting error

**Comment by Ayush0Chaudhary** [comment]
@noman2002 I was not able to mock stream and and I tried other methods, but was unsuccessful.

**Comment by Ayush0Chaudhary** [comment]
@noman2002 @TheHazeEffect please review!

**Comment by noman2002** [comment]
@Ayush0Chaudhary Please don't remove reviewers, it is assigned for a purpose. Please refrain from these type of practice.

**Comment by noman2002** [comment]
> @noman2002 I was not able to mock stream and and I tried other methods, but was unsuccessful.

You can take help from @literalEval

**Comment by palisadoes** [comment]
> @Ayush0Chaudhary Please don't remove reviewers, it is assigned for a purpose. Please refrain from these type of practice.

I 100% agree. Our volunteers have conflicting priorities and may not always be available on demand, but we are all trying to improve the code base.

**Comment by Ayush0Chaudhary** [comment]
Sorry @palisadoes @noman2002 
I tried to ask for re review from Noman and clicked the red icon.

That led to removal of @TheHazeEffect review

It won't happen again, it was not intentional.

**Comment by palisadoes** [comment]
> Sorry @palisadoes @noman2002 I tried to ask for re review from Noman and clicked the red icon.
> 
> That led to removal of @TheHazeEffect review
> 
> It won't happen again, it was not intentional.

Thanks

File: lib/view_model/after_auth_view_models/event_view_models/create_event_view_model.dart [hunk 1]
```diff
@@ -143,6 +143,7 @@ class CreateEventViewModel extends BaseModel {
   Future<void> getImageFromGallery({bool camera = false}) async {
     final _image =
         await _multiMediaPickerService.getPhotoFromGallery(camera: camera);
+    print(_image);
```
Comment (noman2002) [inline lib/view_model/after_auth_view_models/event_view_models/create_event_view_model.dart:146]: Remove print command.

File: lib/view_model/after_auth_view_models/feed_view_models/organization_feed_view_model.dart [hunk 2]
```diff
@@ -73,21 +90,25 @@ class OrganizationFeedViewModel extends BaseModel {
...
-
-    _postsSubscription =
-        _postService.postStream.listen((newPosts) => buildNewPosts(newPosts));
+    _postsSubscription = _postService.postStream.listen((newPosts) {
+      return buildNewPosts(newPosts);
+    });
 
     _updatePostSubscription =
         _postService.updatedPostStream.listen((post) => updatedPost(post));
+    print(_posts);
```
Comment (noman2002) [inline lib/view_model/after_auth_view_models/feed_view_models/organization_feed_view_model.dart:99]: remove print

File: test/widget_tests/after_auth_screens/feed/organization_feed_test.dart [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    // getAndRegisterPostService();
+    // _streamController.add(posts);
+    // await tester.drag(
+    //   find.byType(RefreshIndicator),
+    //   const Offset(0, 200),
+    // );
+
+    // await tester.pumpAndSettle(Duration(milliseconds: 100));
+  });
+}
```
Comment (noman2002) [inline test/widget_tests/after_auth_screens/feed/organization_feed_test.dart:256]: remove these commented code.

**Review by noman2002 — CHANGES_REQUESTED** [review]

**Comment by Ayush0Chaudhary** [comment]
@noman2002 @palisadoes made the suggested changes.

**Review by noman2002 — APPROVED** [review]
Merge this but let the issue be open.

**Review by palisadoes — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**